### ISSUE TITLE
react-native-git-upgrade Error: Cannot find module 'AccessibilityInfo' 

### ISSUE THREAD

**Issue by vitorreis** [facebook/react-native#22040]
- [x] Review the documentation: https://facebook.github.io/react-native
  - [x] Search for existing issues: https://github.com/facebook/react-native/issues
  - [ ] Use the latest React Native release: https://github.com/facebook/react-native/releases

## Environment
``` 
 React Native Environment Info:
    System:
      OS: macOS High Sierra 10.13.6
      CPU: x64 Intel(R) Core(TM) i5-4278U CPU @ 2.60GHz
      Memory: 229.52 MB / 16.00 GB
      Shell: 3.2.57 - /bin/bash
    Binaries:
      Node: 8.9.0 - ~/.nvm/versions/node/v8.9.0/bin/node
      npm: 6.4.1 - ~/.nvm/versions/node/v8.9.0/bin/npm
      Watchman: 4.9.0 - /usr/local/bin/watchman
    SDKs:
      iOS SDK:
        Platforms: iOS 12.1, macOS 10.14, tvOS 12.1, watchOS 5.1
    IDEs:
      Android Studio: 3.2 AI-181.5540.7.32.5056338
      Xcode: 10.1/10B61 - /usr/bin/xcodebuild
    npmPackages:
      react: ^16.6.0-alpha.8af6728 => 16.6.0-alpha.8af6728
      react-native: ^0.57.4 => 0.57.4
    npmGlobalPackages:
      create-react-native-app: 1.0.0
      react-native-cli: 2.0.1
      react-native-git-upgrade: 0.2.7
```
## Description
When I attempt to upgrade my react project to the latest version, the `react-native-git-upgrade` crashes with an error.

I am attempting to upgrade a project created with CRNA -> Ejected in the version RN 0.55.x
```
react-native-git-upgrade
git-upgrade info Check for updates
git-upgrade info Read package.json files
git-upgrade info Check declared version
git-upgrade info Check matching versions
git-upgrade info Check React peer dependency
git-upgrade info Check that Git is installed
git-upgrade info Get information from NPM registry
git-upgrade info Upgrading to React Native 0.57.4, React 16.6.0-alpha.8af6728
git-upgrade info Setup temporary working directory
git-upgrade info Configure Git environment
git-upgrade info Init Git repository
git-upgrade info Add all files to commit
warning: CRLF will be replaced by LF in android/gradlew.bat.
The file will have its original line endings in your working directory.
git-upgrade info Commit current project sources
git-upgrade info Create a tag before updating sources
git-upgrade info Generate old version template
git-upgrade info Add updated files to commit
warning: CRLF will be replaced by LF in android/gradlew.bat.
The file will have its original line endings in your working directory.
git-upgrade info Commit old version template
git-upgrade info Install the new version
npm WARN ajv-keywords@3.2.0 requires a peer of ajv@^6.0.0 but none is installed. You must install peer dependencies yourself.
npm WARN react-redux@5.0.7 requires a peer of react@^0.14.0 || ^15.0.0-0 || ^16.0.0-0 but none is installed. You must install peer dependencies yourself.
npm WARN react-test-renderer@16.3.1 requires a peer of react@^16.0.0 but none is installed. You must install peer dependencies yourself.

git-upgrade info Generate new version template
git-upgrade ERR! An error occurred during upgrade:
git-upgrade ERR! Error: Couldn't find preset "module:metro-react-native-babel-preset" relative to directory "MY_APP_PATH"
    at /Users/v.reis/.nvm/versions/node/v8.9.0/lib/node_modules/react-native-git-upgrade/node_modules/babel-core/lib/transformation/file/options/option-manager.js:293:19
    at Array.map (<anonymous>)
    at OptionManager.resolvePresets (/Users/v.reis/.nvm/versions/node/v8.9.0/lib/node_modules/react-native-git-upgrade/node_modules/babel-core/lib/transformation/file/options/option-manager.js:275:20)
    at OptionManager.mergePresets (/Users/v.reis/.nvm/versions/node/v8.9.0/lib/node_modules/react-native-git-upgrade/node_modules/babel-core/lib/transformation/file/options/option-manager.js:264:10)
    at OptionManager.mergeOptions (/Users/v.reis/.nvm/versions/node/v8.9.0/lib/node_modules/react-native-git-upgrade/node_modules/babel-core/lib/transformation/file/options/option-manager.js:249:14)
    at OptionManager.init (/Users/v.reis/.nvm/versions/node/v8.9.0/lib/node_modules/react-native-git-upgrade/node_modules/babel-core/lib/transformation/file/options/option-manager.js:368:12)
    at compile (/Users/v.reis/.nvm/versions/node/v8.9.0/lib/node_modules/react-native-git-upgrade/node_modules/babel-register/lib/node.js:103:45)
    at loader (/Users/v.reis/.nvm/versions/node/v8.9.0/lib/node_modules/react-native-git-upgrade/node_modules/babel-register/lib/node.js:144:14)
    at Object.require.extensions.(anonymous function) [as .js] (/Users/v.reis/.nvm/versions/node/v8.9.0/lib/node_modules/react-native-git-upgrade/node_modules/babel-register/lib/node.js:154:7)
    at Module.load (module.js:554:32)
git-upgrade ERR! Restore initial sources
```

Can someone offer guidance in how to upgrade the project in this condition? 

# how to reproduce
For example try to upgrade this project: https://github.com/vitorreis/react-native-braze-sdk-app

**Comment by rajivshah3** [comment]
Hmm, this looks similar to #22007. Have you tried the solution suggested there?

**Comment by vitorreis** [comment]
@rajivshah3 thanks for answering... I checked the issue, this happens even without having `.babelrc` file...
You can reproduce it in this with this repo (simple app ejected in 0.55.2) https://github.com/vitorreis/react-native-braze-sdk-app

**Comment by thymikee** [comment]
`react-native-git-upgrade` is now deprecated. Please use automatic `react-native upgrade` if you're on 0.59+ or manual [rn-diff-purge](https://github.com/react-native-community/rn-diff-purge) which `upgrade` uses under the hood
